cfn-resources/util/Makefile

```diff
@@ -0,0 +1,3 @@
+.PHONY: test
+test:
+	go test -v ./...
```

Comment:
Should we move this to the makefile in cfn-resource? the idea is that right now we only have unit tests in the util folder but this may change in the future

Reply:
i think this is ok here as it's handy to create new util functions and being able to test them.  This is a strange project as it has really multiple projects in it. maybe we can have it here and in cfn-resource?

But I agree on the GF action.